django/core/checks/security/csrf.py

```diff
@@ -19,11 +21,28 @@
     id='security.W016',
 )
 
+E024 = Error(
+    'The view function that you have set in CSRF_FAILURE_VIEW setting '
+    'cannot be imported or has an invalid signature.',
+    id='security.E024',
+)
+
 
 def _csrf_middleware():
     return 'django.middleware.csrf.CsrfViewMiddleware' in settings.MIDDLEWARE
 
 
+def _check_csrf_failure_view():
+    from django.middleware.csrf import _get_failure_view
```

Comment:
Does this really need to be an inner import?

Reply:
Yes, otherwise `ImportError` will raise.
`ImportError: cannot import name 'get_callable' from 'django.urls'`